mmv1/templates/terraform/examples/google_storage_managed_folder.tf.erb

```diff
@@ -0,0 +1,10 @@
+resource "google_storage_managed_folder" "managed_folder" {
+  bucket = google_storage_bucket.bucket.name
+  name   = "managed-folder-name"
+}
```

Comment:
Nit: maybe put this after the bucket for clarity?

Reply:
Solved in last commit